plugin/path_impersonated_account.go

```diff
@@ -0,0 +1,259 @@
+package gcpsecrets
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/hashicorp/errwrap"
+	"github.com/hashicorp/vault/sdk/framework"
+	"github.com/hashicorp/vault/sdk/logical"
+)
+
+const (
+	impersonatedAccountStoragePrefix = "impersonated-account"
+	impersonatedAccountPathPrefix    = "impersonated-account"
+)
```

Comment:
Trying to understand, is there a reason why we are declaring two constants with the same value? Can we just rename one to something like `impersonatedAccountPrefix` and use just that one instead?

Reply:
I'm happy to do that but I based this off of https://github.com/hashicorp/vault-plugin-secrets-gcp/blob/3235c5de81b15c9a9db930651be7036e19aaf787/plugin/path_static_account.go#L14-L15

I suspect that it is done like that so that either the storage or the path prefix can be changed without impacting the other. I'm not sure how realistic a change like that is though and if that change did come about splitting the prefixes could be done then.

Reply:
That makes sense. Thank you, I missed that. Since it's following the existing pattern, it's good to leave as is. Thank you again! Great work!